src/api/helpers/encrypt/index.ts

```diff
@@ -11,18 +11,27 @@ const { ENCRYPTION_METHOD, ENCODING, STRING_ENCODING, OUTPUT_ENCODING } = FINANC
  * encrypts data
  * returns encrypted value and initialisation vector
  * @param dataToEncrypt
- * @returns {FinancialDetailsEncryption} value and initialisation vector
+ * @returns {EncryptedData} value and initialisation vector
  */
-const encrypt = (dataToEncrypt: string): FinancialDetailsEncryption => {
+const encrypt = (dataToEncrypt?: string): EncryptedData => {
+  if (!dataToEncrypt) {
+    return {
+      value: '',
+      iv: '',
+    };
+  }
```

Comment:
Do we need this? If we're calling `encrypt`, wouldn't it always have a value?

Can we add `String` type to the documentation and maybe rename `dataToEncrypt` to `str`?

Reply:
If we are wiping the field then it should be set to an empty string